edx_sga/sga.py

```diff
@@ -680,13 +704,13 @@ def upload_allowed(self):
         """
         return not self.past_due() and self.score is None
 
-    def _file_storage_path(self, sha1, filename):
+    def file_storage_path(self, sha1, filename):
```

Comment:
Why was this name changed?

Reply:
because warnings
```W:348,25: Access to a protected member _file_storage_path of a client class (protected-access)
W:360,25: Access to a protected member _file_storage_path of a client class (protected-access)
W:367,25: Access to a protected member _file_storage_path of a client class (protected-access)
W:401,25: Access to a protected member _file_storage_path of a client class (protected-access)
W:408,25: Access to a protected member _file_storage_path of a client class (protected-access)
W:444,25: Access to a protected member _file_storage_path of a client class (protected-access)
W:451,25: Access to a protected member _file_storage_path of a client class (protected-access)
W:490,25: Access to a protected member _file_storage_path of a client class (protected-access)```